src/core/core.ts

```diff
@@ -518,8 +518,7 @@ class Zoomist {
         if (!releasable) {
           e.preventDefault()
         }
-      }
-      else {
+      } else if (!clickable) {
```

Comment:
@fraszczakszymon the name of this option was a bit misleading. I would call it `allowTouchToBubble` or something similar that directly tells what it changes.